src/Console/ConfigurationResolver.php

```diff
@@ -702,16 +702,110 @@ private function validateRules(array $rules): void
         );
 
         if (\count($unknownFixers)) {
+            
+            $renamedRules = [
```

Comment:
Please, could we add new tests to check introduced changes and ensure they won't be broken in the future (due to, for example, some refactoring) ?

Reply:
I push a new commit with tests for every rule. Let me know if it is enough.